pkgs/development/python-modules/easysnmp/default.nix

```
@@ -0,0 +1,72 @@
+{ stdenv
+, lib
+, pkgs
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+
+, python
+, pytest
+, pytestcov
+, pytest-flake8
+, pytest-sugar
+, termcolor
+, ipython
+}:
+
+buildPythonPackage rec {
+  pname = "easysnmp";
+  version = "0.2.5";
+
+  # See https://github.com/kamakazikamikaze/easysnmp/issues/108
+  disabled = !(pythonOlder "3.7");
+
+  src = fetchFromGitHub {
+    owner = "kamakazikamikaze";
+    repo = pname;
+    rev = version;
+    sha256 = "1si9iyxqj6z22jzn6m93lwpinsqn20lix2py3jm3g3fmwawkd735";
+  };
+
+  checkInputs = [
+    pytest
+    pytestcov
+    pytest-flake8
+    pytest-sugar
+    termcolor
+    pkgs.net_snmp
+  ];
+
+  buildInputs = [
+    pkgs.net_snmp
+    pkgs.openssl
+    pkgs.gcc
+    ipython
+  ];
+
+  patches = [./easysnmp.patch];
+
+  buildPhase = ''
+    python setup.py build --basedir=${pkgs.net_snmp}/bin
+  '';
+
+  installPhase= ''
+    dst=$out/${python.sitePackages}
+    mkdir -p $dst
+    export PYTHONPATH="$dst:$PYTHONPATH"
+    python setup.py install --prefix=$out
+  '';
+
+  # I can't get tests to pass, even running by hand. This library needs maintenance.
```

probably want to avoid the first person, and use the passive voice
```suggestion
  # Unable to get tests to pass, even running by hand. The pytest tests have become stale.
```

---

Why?

---

Seems odd reading it when you don't know who is saying it. idk. Don't feel strongly about this one.